style/new_geisha/new_geisha.css

```diff
@@ -489,3 +489,93 @@ h3 {
     width: 100%;
     top: 5px;
 }
+
+/* User Colors */
+.chat_user_name {
+    background: none !important;
+    clip-path: none !important;
+    cursor: pointer;
+    align-self: center;
+    --color-text-shadow: none !important;
+    flex-shrink: 0;
+}
+
+#chatReactContainer .chat_user_name.premiumUser:hover {
```

Comment:
Ah, this is minor atm but I think we should move away from using the ReactContainer class names as that may change when we migrate the pages to an SPA framework. We should set a class name on the root div inside the relevant component (like Chat.js in this case) and attach these styles to that

Reply:
Moved back to chat CSS in commit, accidentally moved on copy/paste.